src/mediasession-helper.js

```diff
@@ -28,23 +38,22 @@ export const EMPTY_METADATA = {
 
 /**
  * Updates the Media Session API's metadata
- * @param {!./service/ampdoc-impl.AmpDoc} ampdoc
- * @param {!./video-interface.VideoMetaDef} metadata
+ * @param {!Window} global
```

Comment:
`win` instead of `global` for consistency (we only use `global` for naming in 3p)